src/wms/src/test/java/org/geoserver/wms/legendgraphic/BufferedImageLegendGraphicOutputFormatTest.java

```diff
@@ -219,6 +222,12 @@ public void testMultipleLayers() throws Exception {
         req.setLayer(ftInfo.getFeatureType());
         req.setStyle(getCatalog().getStyleByName(MockData.ROAD_SEGMENTS.getLocalPart()).getStyle());
 
+        // force antialiasing
+        // note - Mac M1 (ARM mode) does not allow non-antialiased text drawing, so we force to "on"
+        // for consistency
+        // note - Mac M1 (X64 mode) supports non-antialiased text
+        req.setLegendOptions(Collections.singletonMap("fontAntiAliasing", "on"));
```

Comment:
What an odd difference, it must be accidental.

Reply:
ya - kinda weird that you cannot turn off anti-aliasing....